assets/section-main-product.css

```diff
@@ -11,13 +11,18 @@
   margin: 0 auto;
 }
 
+@media screen and (min-width: 750px) {
+  .product--reverse {
+    flex-direction: row-reverse;
+  }
+}
```

Comment:
>I got not strong feelings for this...

Same tbh. The main reasons why I chose to place this here were because 1) there are currently 10 instances of `@media screen and (min-width: 750px) {` within this stylesheet and thought that I really wouldn't be improving things much by doing this (unless I did it for all `@media screen and (min-width: 750px) {` declarations. 2) The current [cascading order of this](https://github.com/Shopify/dawn/blob/c91b37fcb819542350cf4f9be7bd5d6a70afc34c/assets/section-main-product.css#L1-L27) perfectly matches the order of elements in `main-product.liquid` and I didn't want to "break" this. 

I'm totally cool to reposition this under line 28 but I feel like we can improve things a lot more in this stylesheet -- something that could make more sense to tackle later as a refactor perhaps (calibration)?